source/extensions/filters/http/lua/wrappers.cc

```diff
@@ -195,6 +196,43 @@ int StreamInfoWrapper::luaDownstreamSslConnection(lua_State* state) {
   return 1;
 }
 
+int ConnectionStreamInfoWrapper::luaConnectionTypedMetadata(lua_State* state) {
+  const char* filter_name = luaL_checkstring(state, 2);
+
+  const auto& typed_metadata = connection_stream_info_.dynamicMetadata().typed_filter_metadata();
+  const auto it = typed_metadata.find(filter_name);
+
+  if (it == typed_metadata.end()) {
+    lua_newtable(state);
```

Comment:
Would it be better to return a `nil` here to indicate that it wasn't found, instead of an empty table?